c/aws-c-common/.gitignore

```diff
@@ -0,0 +1 @@
+*.c
```

Comment:
`.gitignore` needs to be added to the whitelist of existing files. This could be done by adjusting the general whitelist regexp instead of adding a single exception for this directory.